### DISCUSSION TITLE
Bulk Stencil Library

### DISCUSSION THREAD

**Discussion by InquiredAdvice** [jgraph/drawio#3936] [Q&A]
Is there a way to easily convert a large collection of Visio Shape/Stencil libraries to Draw? Currently I have to do these one at a time by import library, drag and drop file, enter file name, save as xml, then import again to access it.

**Comment by davidjgraph** [comment]
What format are the stencils in?

**Reply by InquiredAdvice** [reply]
most are in .VSS, some are .VSSX. Then i have some icons that are simply SVG files.

**Reply by davidjgraph** [reply]
You could write a powershell to convert all the binary visio formats to the modern XML formats (.vssx). Then you could script those through draw.
